notebooks/geoserver.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.        cleanup_test_data()
</pre><p>Looks like this code will never execute because of how the if/else are defined. Could be removed. </p><p> </p><p>The <code>else</code> could also be removed to do <code>return _resp.json()["resource"]</code> directly.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Maybe not directly apparent, but the code is there mostly for the first 'if' case (or any other case that could be added/modified). If the create attempt returns 409, but the resource failed to be found in the children resources from the parent id, it should go through this code.</p>

Reply:
Ok, sounds good.